package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "hyper-autoprofile",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

```suggestion
  "version": "1.0.2",
```

Can you accept this suggestion to preserve the version number? It will be handled by the release process.